### ISSUE TITLE
Make qpdf availabe on "winget" (Windows Package Manager)

### ISSUE THREAD

**Issue by DNS** [qpdf/qpdf#1144]
https://learn.microsoft.com/en-us/windows/package-manager/package/

So the users can install qpdf directly by typing this on command prompt:

`winget install qpdf`

**Comment by jberkenbilt** [comment]
I don't know if/when I will have to do this, but if someone else takes it on, I'm happy to provide support if it helps. qpdf is already in chocolatey, brew, virtually every Linux distribution, and perhaps other packaging systems as well, and there is documentation for packagers. I maintain the debian packages myself and have made an effort to make it as easy as possible to package qpdf, so hopefully this would be a light lift for someone who is inclined to take it on. This is actually the first time I've heard of winget. It's good to know that it's there.

**Comment by ttimasdf** [comment]
Someone already added `QPDF.QPDF` package to winget package repo (https://github.com/microsoft/winget-pkgs/pull/122994), maybe all we need is to automate the process to create a package manifest and open a PR in a similar way.



Another example PR of automated winget package update by `astral-sh.uv`, https://github.com/microsoft/winget-pkgs/pull/168548

**Comment by petervk** [comment]
I created a pull request for version 11.9.1: https://github.com/microsoft/winget-pkgs/pull/224094

**Comment by petervk** [comment]
QPDF 11.9.1 is not available on winget. This issue can be closed. 

See https://github.com/microsoft/winget-pkgs/pull/224094

**Comment by jberkenbilt** [comment]
Presumably you mean "now" available, not "not" available. I hate that typo, which I sometimes make as well. :-) qpdf 11.10.0 is hopefully coming out this weekend. Is there ongoing action required to keep it updated on winget?

**Comment by petervk** [comment]
Yes, typo, thank you. I neglected to read my message again before posting it.

As for adding the next release it's pretty straight forward. I think I did this one manually before I found this: https://github.com/russellbanks/Komac which makes it very easy. It looks like there is a way to automatically publish it using github tasks if you want to figure that out. I can also take care of adding the next version if you want.

**Comment by petervk** [comment]
I added 11.10.0 to winget-pkgs: https://github.com/microsoft/winget-pkgs/pull/225354

**Comment by jberkenbilt** [comment]
Thanks! Generally I rely on others to update package distribution environments. I do debian myself, but other people do all the rest. Thanks for taking care of winget.
